tests/test_createApFakes.py

```diff
@@ -54,6 +58,43 @@ def setUp(self):
                             + int(self.nSources * self.fraction))
         self.rng = np.random.default_rng(1234)
 
+    def testRunQuantum(self):
+        """Test the run quantum method with a gen3 butler.
+        """
+        root = tempfile.mkdtemp()
+        dimensions = {"instrument": ["notACam"],
+                      "skymap": ["deepCoadd_skyMap"],
+                      "tract": [0, 42],
+                      }
+        testRepo = butlerTests.makeTestRepo(root, dimensions)
+        fakesTask = CreateRandomApFakesTask()
+        connections = fakesTask.config.ConnectionsClass(
+            config=fakesTask.config)
+        butlerTests.addDatasetType(
+            testRepo,
+            connections.skyMap.name,
+            connections.skyMap.dimensions,
+            connections.skyMap.storageClass)
+        butlerTests.addDatasetType(
+            testRepo,
+            connections.fakeCat.name,
+            connections.fakeCat.dimensions,
+            connections.fakeCat.storageClass)
+
+        dataId = {"skymap": "deepCoadd_skyMap", "tract": 0}
+        butler = butlerTests.makeTestCollection(testRepo)
+        butler.put(self.simpleMap, "deepCoadd_skyMap", {"skymap": "deepCoadd_skyMap"})
+
+        quantum = testUtils.makeQuantum(
+            fakesTask, butler, dataId,
+            {key: dataId for key in {"skyMap", "fakeCat"}})
+        run = testUtils.runTestQuantum(fakesTask, butler, quantum, True)
+        # Actual input dataset omitted for simplicity
+        run.assert_called_once()
```

Comment:
I skipped this in the example because it would have added a lot of distracting boilerplate, but since you have `self.simpleMap` and the tract ID, you should be able to predict the inputs to `run` and call `assert_called_once_with`.